cylc/flow/scheduler.py

```diff
@@ -276,7 +277,7 @@ async def install(self):
 
         """
         # Install
-        source = suite_files.get_suite_source_dir()
+        source = suite_files.get_workflow_source_dir(Path.cwd())
```

Comment:
`get_workflow_source_dir()` now also returns the source path. This was to enable more informative error messages.